Library/Homebrew/lock_file.rb

```diff
@@ -54,7 +55,7 @@ def lock
             raise OpenFileChangedOnDisk
           end
 
-          @lockfile = lockfile
+          @lockfile = T.let(lockfile, T.nilable(File))
```

Comment:
```suggestion
```

Reply:
Might need to keep it as
```ruby
@lockfile = lockfile
```
i.e. drop only the `T.let` bit.